tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -118,26 +118,35 @@ def try_client_handshake(endpoint, arguments, expected_cipher):
     """
     s2nc_cmd = ["../../bin/s2nc", "-f", "./trust-store/ca-bundle.crt", "-a", "http/1.1"] + arguments + [str(endpoint)]
     currentDir = os.path.dirname(os.path.realpath(__file__))
-    s2nc = subprocess.Popen(s2nc_cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, cwd=currentDir)
+    s2nc = subprocess.Popen(s2nc_cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, cwd=currentDir, universal_newlines=True)
 
-    found = 0
-    expected_output = "Cipher negotiated: "
-    if expected_cipher:
-        expected_output += expected_cipher
+    try:
+        outs, errs = s2nc.communicate(timeout=5)
 
-    for line in range(0, NUM_EXPECTED_LINES_OUTPUT):
-        output = str(s2nc.stdout.readline().decode("utf-8"))
-        if expected_output in output:
-            found = 1
-            break
+        found = 0
+        expected_output = "Cipher negotiated: "
+        if expected_cipher:
+            expected_output += expected_cipher
 
-    s2nc.kill()
-    s2nc.wait()
+        for line in range(0, NUM_EXPECTED_LINES_OUTPUT):
+            output = str(outs)
+            print(output, end='')
```

Comment:
This is going to clog up our integ test output. The output for our integration tests should just be a failure/success for each test, otherwise its a lot of info.

Reply:
Thats fair, but also quite nice for debugging when things go wrong. Otherwise each time it breaks, you need to enable this and thats extra effort.

I can see it going both ways.. Thinking that Ill keep the `Connecting to: kms.us-east-1.amazonaws.com        Attempt:1 `
for now and remove the others.

Reply:
removed some `print` in https://github.com/aws/s2n-tls/pull/3148/commits/ebb53b03d45af80b18a7be272cbc004b12cd7057